.pre-commit-config.yaml

```diff
@@ -46,25 +34,3 @@ repos:
                 - types-requests
                 - types-tabulate
                 - types-toml
-
-    - repo: https://github.com/asottile/setup-cfg-fmt
-      rev: v2.5.0
-      hooks:
-          - id: setup-cfg-fmt
-
-    - repo: https://github.com/pycqa/isort
-      rev: 5.13.2
-      hooks:
-          - id: isort
-            args: [--profile, black, --filter-files]
-
-    - repo: https://github.com/sirosen/check-jsonschema
```

Comment:
This is checking the GHA yaml file against publicly available jsonschema. But the jsonschema is downloaded from the internet, so this check is slow.

Reply:
Let's not remove the useful checks that are slow. I looked at the last commit - the maximum time I saw was two minutes.

While I am happy about the performance improvement, I would like it not to be at the cost of reducing the robustness.

Reply:
I am going to push back a little bit: as I understand it, this checks the correctness of the GHA yaml file against the specification. So it helps you catch issues when you change the file *before* you push to GitHub. 

But I don't think it reduces the robustness -- if you make a mistake, GitHub actions will yell at you anyway. I think by definition it cannot catch anything that GHA would not catch.

Reply:
good point, agreed